front/app/containers/Admin/projects/project/analysis/FilterItems/index.tsx

```diff
@@ -1,138 +1,140 @@
+    },
+    votes_to: {
+      translationKey: 'Votes',
+      predicate: '<',
+    },
+    comments_from: {
+      translationKey: 'Comments',
+      predicate: '>',
+    },
+    comments_to: {
+      translationKey: 'Comments',
+      predicate: '<',
+    },
   };
 
-  const filterEntries = Object.entries(filters)
-    .filter(([key]) => key !== 'tag_ids')
-    .filter(([key]) => !(isEditable && key === 'search'));
-
-  const filterItemDisplayValue = (
-    key: string,
-    value: string | string[] | undefined | number
-  ) => {
-    switch (key) {
-      case genderUrlQueryParamKey:
-        return (
-          isArray(value) &&
-          value.map((value: string) => (
-            <Box key={value} ml="4px">
-              {localize(
-                genderOptions?.data.find((option) => {
-                  return option.attributes.key === value;
-                })?.attributes.title_multiloc
-              )}
-            </Box>
-          ))
-        );
-      case domicileUrlQueryParamKey:
-        return (
-          isArray(value) &&
-          value.map((value: string) => (
-            <Box key={value} ml="4px">
-              {localize(
-                domicileOptions?.data.find(
-                  (option) => option.attributes.key === value
-                )?.attributes.title_multiloc
-              )}
-            </Box>
-          ))
-        );
-
-      default:
-        return value;
-    }
-  };
+  const filterEntries = Object.entries(filters).filter(
+    ([key]) => key !== 'tag_ids'
+  );
 
   return (
     <Box display="flex" flexWrap="wrap" gap="4px">
-      {filterEntries.map(([key, value]) => (
-        <Box
-          key={key}
-          py="4px"
-          px="8px"
-          borderRadius={stylingConsts.borderRadius}
-          borderColor={colors.success}
-          borderWidth="1px"
-          borderStyle="solid"
-          bgColor={colors.white}
-          color={colors.success}
-          display="flex"
-        >
-          {translationKeys[key]}
-          {filterItemDisplayValue(key, value)}
-
-          {isEditable && (
-            <IconButton
-              iconName="close"
-              iconColor={colors.success}
-              iconColorOnHover={colors.success}
-              iconWidth="16px"
-              iconHeight="16px"
-              onClick={() => {
-                removeSearchParams([key]);
-              }}
-              a11y_buttonActionMessage="Remove filter"
+      {filterEntries.map(([key, value]) => {
+        const [_fullKey, subject, customFieldId, _postfix, clause] =
+          key.match(/^(author|input)_custom_([a-f0-9-]+)(_(from|to))?$/) || [];
+        const predicate = clauseToPredicate(clause);
+        if (subject === 'input') {
+          return (
+            <InputFieldFilterItem
+              key={key}
+              customFieldId={customFieldId}
+              filterKey={key}
+              filterValue={value}
+              predicate={predicate}
+              isEditable
```

Comment:
Should this component always be editable? Or should it pass on the prop passed to `FilterItems`?

Reply:
It should definitly pass down the prop, indeed, thanks.

I wasn't able to test this in summarization due to my local db not being compatbible the the BE code anymore, so missed it.